### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tylertomlinson**
Creating README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,55 @@
-# Monster Shop
-BE Mod 2 Week 4/5 Group Project
+<p align="center">
+  <img width="500" height="200" src="https://media-exp1.licdn.com/dms/image/C4D1BAQHZ3m-yGm8cvA/company-background_10000/0?e=2159024400&v=beta&t=4R94SSaU7ug1AXLuYJzmSsqKUUdseWG-BlfpnZiwnKI">
+</p>
```
Comment (alex-latham) [inline README.md:3]: Can you render HTML in a .md file?

Comment (alex-latham) [reply README.md:3]: `Monster Shop Logo`

Comment (alex-latham) [reply README.md:3]: Oh nevermind! I just previewed it. Genius.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+| Admin         | admin@example.com | password_admin  |
+| Merchant Employee | merchant@example.com     | password_merchant |
+| Standard User | user@example.com     | password_regular |
+
+
+# Monster Shop
+BE Mod 2 Week 4/5 Group Project
+
+
 Students will be put into 3 or 4 person groups to complete the project.\n
```
Comment (alex-latham) [inline README.md:53]: We could probably just drop everything from 53 on

**Review by danielpselph — APPROVED** [review]
Great work on the readme, thank you for submitting this.

**Pull request merged**